main.tf

```diff
@@ -0,0 +1,245 @@
+/**
```

Comment:
Why is this `main.tf` checked into the top-level directory?  Why not in the `integration_tests` directory?

Reply:
`main.tf` in the top directory has the resources that needs to be created to deploy Forseti. `main.tf` in `integration_tests/fixtures/forseti` has the resources that needs to be created to run tests.

Reply:
`main.tf` in the root directory has been removed.